src/style.css

```diff
@@ -1,3 +1,62 @@
+html {
+  font-family: "Roboto", sans-serif;
+}
+
 body {
   margin: 0;
+  line-height: 1;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+}
+
+.logo__img {
+  width: 40px;
+}
+
+.nav__list {
+  display: flex;
+  white-space: nowrap;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__item {
+  list-style: none;
+  margin-right: 20px;
+  position: relative;
+}
+
+.nav__item:last-child {
+  margin-right: 0;
+}
+
+.nav__link {
+  display: block;
+  color: inherit;
+  text-decoration: none;
+  text-transform: uppercase;
+  font-size: 12px;
+  font-weight: 500;
+  padding: 24px 0;
+}
+
+.nav__link:hover,
+.is-active {
+  color: rgba(0, 172, 220, 1);
```

Comment:
* Consider using a hex value here
* Consider using variables for repeating colors